pkg/cache/snapshot.go

```diff
@@ -128,6 +129,7 @@ func (c *ClusterQueue) snapshot() *ClusterQueue {
 		AllocatableResourceGeneration: c.AllocatableResourceGeneration,
 		Usage:                         make(FlavorResourceQuantities, len(c.Usage)),
 		Workloads:                     make(map[string]*workload.Info, len(c.Workloads)),
+		GuaranteedQuota:               make(FlavorResourceQuantities, len(c.GuaranteedQuota)),
```

Comment:
```suggestion
```

IIUC, no need to initialize `GuranteedQuota` here since we initialize this in Line 151, right?

Reply:
Agree.